crates/bevy_gltf/src/loader/gltf_ext/mesh.rs

```diff
@@ -0,0 +1,42 @@
+use bevy_render::mesh::PrimitiveTopology;
+
+use gltf::mesh::{Mesh, Mode, Primitive};
+
+use crate::GltfError;
+
+pub trait MeshExt {
```

Comment:
As I said before, I think we should just avoid these extension traits. I'd much prefer just free functions, like `primitive_name_for_mesh(mesh: &gltf::Mesh, primitive: &Primitive)`. (Similar for the other extension traits)